hicommonwealth/commonwealth
# Removed `FLAG_MY_COMMUNITY_STAKE_PAGE_ENABLED` flag

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/7354

## Description of Changes
- Removed `FLAG_MY_COMMUNITY_STAKE_PAGE_ENABLED` feature flag

## "How We Fixed It"
N/A

## Test Plan
- Visit `/myCommunityStake` page, verify it loads
- Open profile menu, verify you see the `My community stake` option

## Deployment Plan
Merge After MyCommunityStakes page is enabled in production

## Other Considerations
N/A
